resources/contenttweaker/lang/en_us.lang

```diff
@@ -1 +1,5 @@
-item.contenttweaker.printed_parallel.name=Printed Parallel Circuit
\ No newline at end of file
+item.contenttweaker.printed_parallel.name=Printed Parallel Circuit
+item.contenttweaker.coin_bronze.name = Bronze Coin (Steam)
+item.contenttweaker.coin_steel.name = Steel Coin (LV)
+item.contenttweaker.coin_aluminium.name = Aluminium Coin (MV)
+item.contenttweaker.coin_stainless_steel.name = Stanless Steel Coin (V)
```

Comment:
Typo: "Stanless"
Should say "Stainless Steel Coin (HV)"
Also each of these lang entries have an space before and after the `=` - this WILL add a space at the beginning of the item names, and may cause some to also not be translated. There should be NO spaces before and after the equals sign on either side.

Reply:
Fixed